### ISSUE TITLE
More detailed build instructions

### ISSUE THREAD

**Issue by winder** [pianobooster/PianoBooster#260]
I'm setting up some build automation with github actions, but am having trouble with the build process. If you could provide more detailed instructions about how to build, that would be very helpful.

Specifically it's been difficult to find the dependencies, and release package commands.

What I have so far:

Linux (build works, but I don't know how to make the AppImage)
```
sudo apt-get install build-essential cmake pkg-config libftgl-dev qtbase5-dev qttools5-dev librtmidi-dev flu    id-soundfont-gm libfluidsynth-dev
mkdir build
cd build
cmake ..
make
```

Mac (not working)
```
brew install cmake qt5 ftgl jack fluidsynth pkg-config
mkdir build
cd build
cmake .. -DCMAKE_PREFIX_PATH=$(brew --prefix qt)
make

failed to build until updating CMakeLists.txt with
"include_directories (${FLUIDSYNTH_INCLUDE_DIRS})"

then fails to link:
ld: warning: directory not found for option '-Lfluidsynth'
ld: library not found for -lfluidsynth
```

Windows - I haven't started this yet.

**Comment by louis-barman** [comment]
`I'm setting up some build automation with github actions` Oh wow that really is fantastic and would be so useful thank you very much for starting this. 

However I am am just getting back onto completing the release 0.7.4 which is my highest priority  at the moment and then I need to update the website with details of this new release. After the release pianobooster 0.7.4 I am planning to move the official repo from here to https://github.com/pianobooster . And so that would be the best place to setup the build automation.

I have just raised an issue regrading automation over there please see https://github.com/pianobooster/PianoBooster/issues/1 and it would be best to continue our conversation  in that issue.

**Comment by louis-barman** [comment]
Fixed in v1.0.0
